lib/orchestrator/process_job.rb

```diff
@@ -41,6 +41,10 @@ def update_dynamodb
     end
 
     def inform_spi
+      RestClient.patch(
```

Comment:
It looks like nothing is really patched, so this is just like a "hey there's been an update" call?

Reply:
Yeah - which then triggers the website to process the update and then further update the record to state it's been processed. I wasn't sure what the right verb/pattern for that was tbh. Thoughts?

Reply:
I think PATCH makes the most sense in this case, just wanted to check that I understood it correctly.

Reply:
OK, sweet :)